src/app/core/components/form-element/form-element.component.html

```diff
@@ -0,0 +1,17 @@
+<div [ngSwitch]="element.name" style="margin: 10px; text-align: center">
```

Comment:
no inline styles
replace them to SCSS
```suggestion
<div [ngSwitch]="element.name">
```